index.html

```diff
@@ -90,4 +90,13 @@ <h3 class="post-subtitle">
      </article>
     {% endfor %}
   </div>
+
+  <!-- 2020 fundraiser sponsor logos -->
+  <div id="fundraiser">
+	  <h2 style="font-size:42px;">Supporters</h2>
+	  <p>Bitcoin ABC relies on sponsorships to fund all its work, including devops, maintenance, backports, development, release management, technical support and more. Become a supporter today at <a target=_fund" href="https://fund.bitcoinabc.org/">fund.bitcoinabc.org</a>.</p>
+	  <p><a target="_sponsor1" href="https://fund.bitcoinabc.org/"><img src="img/sponsors/sponsor.png" height="100" width="500" alt="Sponsor Bitcoin ABC today at fund.bitcoinabc.org" /></a></p>
```

Comment:
Since it's the same link as above, the target should probably also be `"_fund"`

Reply:
The patch did not apply against index.html so applied it manually. Thanks for your help.